Source/ending.py

```diff
@@ -0,0 +1,96 @@
+# Author: Siena Okuno
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Determines endings
+
+import pygame
+import sys
+
+def screenScroll(screen, color):
+    '''Turns screen a color by filling in with rectangles.'''
+    for i in range(700):
+        pygame.draw.rect(screen, color, (0,i,1000,i+1))
+        i+=1
+        pygame.display.flip()
+
+
+def ending(victory, levels, screen):
+    '''Creates basic, text-only ending'''
+    font50 = pygame.font.Font(None, 50)
+    font20 = pygame.font.Font(None, 20)
+    if victory == 0 or victory == 2:
+        screenScroll(screen, (0,0,0))
+        screen.blit(font50.render("Looks like your portfolio", 1, (255,255,255)), (300, 250))
+        screen.blit(font50.render("could use some improvement", 1, (255,255,255)), (280, 350))
+        Done = False
+        while not Done:
+            for event in pygame.event.get():
+                if event.type == pygame.QUIT:
+                    Done = True
+            for i in range(len(levels)):
+                screen.blit(font20.render(levels[i], 1, (200,200,200)), (100, 450+i*25))
+            pygame.display.flip()
+    if victory == 1:
```

Comment:
It looks like `victory` is used a boolean? If so, prefer the boolean type: `True`/`False`, instead of `1`/`0`. This is clearer in intent.